docs/packages/pkg/Fruit.rst

```diff
@@ -0,0 +1,43 @@
+.. spelling::
+
+    Google Fruit
```

Comment:
Package is called `Fruit` so I guess it should be just `Fruit` instead of `Google Fruit`.